index.js

```diff
@@ -175,7 +175,8 @@ async function installBundler(bundlerVersionInput, platform, rubyPrefix, engine,
   var bundlerVersion = bundlerVersionInput
 
   if (bundlerVersion === 'default' || bundlerVersion === 'Gemfile.lock') {
```

Comment:
Does it make sense still to allow `Gemfile.lock` as an option to the `bundler` configuration? Would it be too hard to make it deprecated?

The hard question is: what should happen when someone uses the `Gemfile.lock` configuration AND has a BUNDLE_GEMFILE env variable?

Reply:
The idea is if we change the default we'd still have a way to express the previous default.
So I think `Gemfile.lock` is good enough (`foo/bar.lock` is still "a lockfile for a "Gemfile"), maybe `lockfile` would be better but it seems less intuitive and not worth it.